guides/common/modules/con_introduction-to-dns-dhcp-and-tftp-integration-in-project.adoc

```diff
@@ -0,0 +1,5 @@
+[id="introduction-to-dns-dhcp-and-tftp-integration-in-{Project}_{context}"]
+= Introduction to DNS, DHCP, and TFTP integration in {Project}
+
+By default, {Project} uses its integrated DNS, DHCP, and TFTP services. However, a key of {Project} is the ability to seamlessly integrate with existing network services. By configuring the corresponding providers, you can use existing DNS, DHCP, and TFTP services and integrate them in to {Project}.
```

Comment:
By default it doesn't have any DNS, DHCP or TFTP service configured. It is required to configure one. It does make it easier to enable each service where the installer manages the service.

Reply:
I rewrote the intro.